packages/contracts/package.json

```diff
@@ -84,6 +84,7 @@
     "@openzeppelin/contracts": "4.8.1",
     "@openzeppelin/contracts-upgradeable": "4.8.1",
     "@aragon/osx-v1.0.1": "npm:@aragon/osx@1.0.1",
+    "@aragon/osx-v1.3.0-rc0.2": "npm:@aragon/osx@1.3.0-rc0.2",
```

Comment:
We must remember to bump this to the final release later on
Still not having a release v1.3.0 is becoming a problem.